Check4XorgX11Devel.py

```diff
@@ -0,0 +1,46 @@
+#############################################################################
+# File          : Check4XorgX11Devel.py
+# Package       : rpmlint
+# Author        : Tomas Chvatal
+# Created on    : Thu Feb 08 14:00:13 CEST 2018
+# Purpose       : Check for obsolete xorg-x11-devel dependency
+#############################################################################
+
+from Filter import addDetails, printError
+import AbstractCheck
+import Config
+
+xorg_devel_tag = 'suse-obsolete-xorg-x11-devel-requirement'
+
+
+class Check4XorgX11Devel(AbstractCheck.AbstractFilesCheck):
+    def __init__(self):
+        self.map = []
+        AbstractCheck.AbstractCheck.__init__(self, 'Check4Xinetd')
+
+    def check(self, pkg):
+        # Do not skip sourcepackage where it seems requires are also BRs.
+        # If package has any of Requires or BuildRequires then fail it
+        for req in pkg.requires() + pkg.prereq():
```

Comment:
so here you're checking runtime requires

Reply:
but pkg.requires() on source package based on the code should be checking the BuildRequires too.

Reply:
@dirkmueller well I suppose you are right, but then how does one check BuildRequires using rpmlint, or that is not possible?

Reply:
probably best to limit the check with ```pkg.isSource()``` - so we check Requires of a .src.rpm (which is equivalent to BuildRequires of the .spec); since RPMLint operates on RPMs, that seems like the best thing to do